arch/arm/src/stm32h7/hardware/stm32h7x3xx_memorymap.h

```diff
@@ -60,10 +60,18 @@
 
 #define STM32_DTCRAM_BASE    0x20000000     /* 0x20000000-0x2001ffff: DTCM-RAM on TCM interface */
 #define STM32_AXISRAM_BASE   0x24000000     /* 0x24000000-0x247fffff: System AXI SRAM */
-#define STM32_SRAM1_BASE     0x30000000     /* 0x30000000-0x3001ffff: System SRAM1 */
-#define STM32_SRAM2_BASE     0x30020000     /* 0x30020000-0x3003ffff: System SRAM2 */
-#define STM32_SRAM3_BASE     0x3004c000     /* 0x30040000-0x30047fff: System SRAM3 */
-#define STM32_SRAM123_BASE   0x30000000     /* 0x30000000-0x30047fff: System SRAM123 */
+
+#ifdef CONFIG_ARCH_CHIP_STM32H7_CORTEXM7
```

Comment:
@raiden00pl Why is this here? Would you please explain this? Where are the second set of addresses from?

Reply:
From RM0399 -> 2.4 Embedded SRAM:
```
The AHB SRAMs of the D2 domain are also aliased to an address range below
0x2000 0000 to maintain the Cortex®-M4 Harvard architecture:
•AHB SRAM1 also mapped at address 0x1000 0000 and accessible by all system
masters through D2 domain AHB matrix
•AHB SRAM2 also mapped at address 0x1002 0000 and accessible by all system
masters through D2 domain AHB matrix
•AHB SRAM3 also mapped at address 0x1004 0000 and accessible by all system
masters through D2 domain AHB matrix
```

Reply:
@raiden00pl is it possible to put that as a comment near the code? In the future, it will be easier to understand.

Reply:
done